test/multinomial.jl

```diff
@@ -207,3 +207,9 @@ p_v = [0.1, 0.4, 0.3, 0.8]
 @test_throws ArgumentError Multinomial(10, p_v)
 @test_throws ArgumentError Multinomial(10, p_v; check_args=true)
 Multinomial(10, p_v; check_args=false) # should not warn
+
+# check different prob vector types
+p = [0.2, 0.4, 0.3, 0.1]
+@test (rand(Multinomial(10, p)); true)
+@test (rand(Multinomial(10, convert.(Float32, p))); true)
+# @test (rand(Multinomial(10, convert.(ForwardDiff.Dual{Float64}, p))); true)
```

Comment:
```suggestion
```

Reply:
I had left it in to revisit the issue in a different issue/PR, but yeah makes more sense to take it out.